src/Interceptor/Impl/CacheInterceptor.php

```diff
@@ -256,12 +286,12 @@ private function guessObjectsTags(mixed $object, array $excludedClasses = []): a
                 continue;
             }
             $subObject = $this->getPropertyValue($ref, $object, $propRef->getName());
-            if (\is_iterable($subObject)) {
+            if (is_iterable($subObject)) {
```

Comment:
we should keep the leading backslash

Reply:
it seems it is not included by NativeFunctionInvocationFixer (its default configuration is @compiler_optimized and it removes this backslack 🤷 )